api/src/test/java/org/openmrs/util/DatabaseUpdaterDatabaseIT.java

```diff
@@ -30,7 +30,7 @@
 	 * 
 	 * This constant needs to be updated when adding new Liquibase update files to openmrs-core.
 	 */
-	private static final int CHANGE_SET_COUNT_FOR_2_1_X = 863;
```

Comment:
Did we drop a changeset?

Reply:
The generated core-data snapshots contained inserts for the global_property table. The actual data was added to the database by Java classes and not by the original Liquibase change logs. So that change set was removed again.

Reply:
Oh i see! 😊